src/config_file.c

```diff
@@ -284,8 +284,6 @@ static int config_set(git_config_backend *cfg, const char *name, const char *val
 	if ((error = config_write(b, name, key, NULL, esc_value)) < 0)
 		goto out;
 
-	error = config_refresh(cfg);
```

Comment:
Commit message says "cocked" instead of "cached".

Reply:
Oops, fixed!